composer.json

```diff
@@ -4,7 +4,7 @@
     "type": "drupal-module",
     "license": "GPL-2.0-or-later",
     "require": {
-        "dpc-sdp/tide_core": "^1.2.6",
+        "dpc-sdp/tide_core": "dev-feature/SDPA-4582-display-h3-headings as 1.6.9",

```

Comment:
Need to be removed before merging and after tide_core release for the new field storage.